### DISCUSSION TITLE
Achievement milestones at non-round numbers

### DISCUSSION THREAD

**Discussion by smichel17** [streetcomplete/StreetComplete#2700] [Feedback]
I would expect to get achievements at round numbers, but I frequently get them at seemingly-arbitrary numbers. Ex:

![image](https://user-images.githubusercontent.com/6563664/112998960-0dd5c980-915e-11eb-88d8-94e040e65b8d.png)

I don't really mind it — it makes the achievements seem more like things that randomly pop up every so often than goals I'm working towards, which may even be good — but it does seem odd, so I wonder if it's a bug.

**Comment by matkoniecz** [comment]
I was also surprised, but right now achievements are simply defined in such way.

I admit that I considered making PR changing that but was not bothered sufficiently to implement anything.

```
            R.string.achievement_surveyor_solved_X,
            TotalSolvedQuests,
            // levels: 10, 30, 60, 100, 150, 210, 280, 360, 450, 550, 660, 780, 910, 1050, ...
            { lvl -> (lvl + 1)*10 },
```

Or in this specific case: https://github.com/streetcomplete/StreetComplete/blob/60653613c063fbeddf1d8508f68c89f4a1e57f04/app/src/main/java/de/westnordost/streetcomplete/data/user/achievements/AchievementsModule.kt#L385 - you get new level every 4 days of mapping
